clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1000,6 +987,14 @@ public void run() {
             pushJobSetting.rewindTimeInSecondsOverride = DEFAULT_RE_PUSH_REWIND_IN_SECONDS_OVERRIDE;
             LOGGER.info("Overriding re-push rewind time in seconds to: {}", DEFAULT_RE_PUSH_REWIND_IN_SECONDS_OVERRIDE);
           }
+          if (pushJobSetting.repushTtlInHours != NOT_SET) {
+            // the schema path will be suffixed by the store name, e.g. /tmp/veniceRmdSchemas/<store_name>
```

Comment:
Should we cleanup the path once we are done?

Reply:
My current idea is when fetching all schemas from controller, if the schema has persisted, we will skip this, as one minor optimization, so the cleanup is not needed.

Ref:
[code](https://github.com/linkedin/venice/blob/4e510ce4cb349908b2a0eb8dbfee9825a5dcc673/clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/schema/HDFSRmdRmdSchemaSource.java#L107)

Reply:
Not sure if we should trust prior runs's temp dir. It could have been corrupted. Is it expensive to repopulate every time ?

Reply:
> It could have been corrupted.

That's a valid point. One way to workaround it is adding permissions so that only venice can modify these files.
>  Is it expensive to repopulate every time ?

We don't have data on how expensive it is yet as it depends on how RMD schemas a store can have.
I think for the time being, we can add the logic to delete the it after the push job is done and tweak this logic as we go. Will do it at the next revision.